### PR TITLE
Add missing tax/vat below price

### LINKED ISSUE #612: Product Page: Tax/VAT indication after price missing
**Describe the current behavior**
incl./excl. Vat, excl Shipping and link to shipping policies missing behind or below product price.

**Describe the expected behavior**
After or below price should be indicated if Tax / VAT is included or excluded and same for shipping with link to policies.
This is in all free Shopify Themes present, but missing in dawn and mandatory requirement in EU especially german markets.

**Version information (Dawn, browsers and operating systems)**
- Dawn Version: 1.0.0
- Chrome Version 91.0.4472.114
- macOS Version 11.3.1

**Possible solution**
adding code to product-template.liquid to display the missing text and link

### PR DESCRIPTION
**Pull request by sofiamatulis**
**Why are these changes introduced?**

Fixes #612.

**What approach did you take?**

I didn't need to add this to cart because it was already there: https://github.com/Shopify/dawn/blob/main/sections/main-cart-footer.liquid#L42-L52

I added it to the product page and saw in Debut we had it at the featured product section so I added it there as well

**Other considerations**

**Demo links**

- [Editor](https://os2-demo.myshopify.com/admin/themes/126386733078/editor)

**Checklist**
- [ ] Followed [theme code principles](https://github.com/Shopify/dawn/blob/main/.github/CONTRIBUTING.md#theme-code-principles)
- [ ] Linted with [Theme Check](https://github.com/Shopify/theme-check)
- [ ] Tested on [mobile](https://shopify.dev/themes/store/requirements#mobile-browser-requirements)
- [ ] Tested on [multiple browsers](https://shopify.dev/themes/store/requirements#desktop-browser-requirements)
- [ ] Tested for [accessibility](https://shopify.dev/themes/best-practices/accessibility)

### CODE REVIEW

File: sections/featured-product.liquid [hunk 1]
```diff
@@ -89,6 +89,16 @@
                   {%- endform -%}
                 </div>
               {%- endif -%}
+              {%- if shop.taxes_included or shop.shipping_policy.body != blank -%}
+                <div class="caption rte">
```
Comment (tauthomas01) [inline sections/featured-product.liquid:93]: When there are no product image, the text is left aligned. Can we align it center?
When there is a product image, it looks fine.

[alt](https://screenshot.click/14-27-cy194-fwz3l.png)

**Review by tauthomas01 — COMMENTED** [review]
Looks good 👍 I just found a small detail on the visual, but functionality-wise it works.

image

File: sections/featured-product.liquid (hunk 1 above)
Comment (sofiamatulis) [reply sections/featured-product.liquid]: Good catch! Fixed https://github.com/Shopify/dawn/pull/629/commits/e5c9de9ce6e5626bddfa788e2112fcc937234fce

**Comment by ludoboludo** [comment]
You can probably request translation as well since I don't think the text will change :)

File: assets/section-main-product.css [hunk 2]
```diff
@@ -342,6 +351,10 @@ a.product__text {
   text-align: center;
 }
 
+.product--no-media .product__tax {
+  text-align: center;
+}
```
[the remark this answers is not in the document]
Comment (sofiamatulis) [reply assets/section-main-product.css:347]: @ludoboludo I tried that but the `flex` makes the spacing too tight.
https://screenshot.click/15-06-s1ly2-zkt09.mp4

Reverted it back

[the remark this answers is not in the document]
Comment (ludoboludo) [reply assets/section-main-product.css:347]: Oops I meant on line 322 not 332. Where it applies `text-align: center;` only.

File: sections/main-product.liquid [hunk 3]
```diff
@@ -93,6 +93,16 @@
                 {{ form | payment_terms }}
               {%- endform -%}
             </div>
+            {%- if shop.taxes_included or shop.shipping_policy.body != blank -%}
```
[the remark this answers is not in the document]
Comment (sofiamatulis) [reply sections/main-product.liquid:96]: Updated it to be above to follow the same we have in Debut 👍

File: sections/featured-product.liquid [hunk 4]
```diff
@@ -89,6 +89,16 @@
                   {%- endform -%}
                 </div>
               {%- endif -%}
+              {%- if shop.taxes_included or shop.shipping_policy.body != blank -%}
```
[the remark this answers is not in the document]
Comment (sofiamatulis) [reply sections/featured-product.liquid:92]: Updated it to follow the order we have in Debut 👍

**Review by ludoboludo — APPROVED** [review]
Looks good 👍  I can re approve once the translations are back.

**Review by melissaperreault — APPROVED** [review]
LGTM! Thanks Sofia!

**Review by ludoboludo — APPROVED** [review]

**Review by melissaperreault — APPROVED** [review]

**Pull request merged**